### PR TITLE
Initial Pull Request

### PR DESCRIPTION
**Pull request by fernandohds564**
image

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+develop: clean uninstall
+	sudo -H pip-sirius install --no-deps -e ./
+
+uninstall:
+ifneq (,$(wildcard $(EGGLINK)))
+	sudo rm -r $(EGGLINK)
+endif
+ifneq ($(ISINST),0)
+	sudo -H pip-sirius uninstall -y $(PACKAGE)
+	sudo sed -i '/pyqt-apps/d' $(DISTPATH)/easy-install.pth
```
Comment (anacso17) [inline Makefile:25]: I think we do not need `pyqt-apps` here

Comment (fernandohds564) [reply Makefile]: right!
fixed!

**Review by murilobalves — APPROVED** [review]

**Review by xresende — APPROVED** [review]

**Review by murilobalves — APPROVED** [review]

**Review by anacso17 — APPROVED** [review]

**Review by anacso17 — APPROVED** [review]

File: siriushlafac/as_ap_trajfit/main.py [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+            self.wid_nuy.setValue(14.15)
+        else:
+            self.wid_nux.setValue(19.204)
+            self.wid_nuy.setValue(7.314)
+        self.wid_nux.setSingleStep(0.001)
+        self.wid_nuy.setSingleStep(0.001)
+        self.wid_nux.setDecimals(4)
+        self.wid_nuy.setDecimals(4)
+
+        wid.layout().addWidget(QLabel('\u03bd<sub>y</sub>', wid), 1, 0)
```
Comment (murilobalves) [inline siriushlafac/as_ap_trajfit/main.py:146]: I think it is <sub>x</sub> here

**Review by murilobalves — CHANGES_REQUESTED** [review]

File: siriushlafac/as_ap_trajfit/main.py (hunk 2 above)
Comment (fernandohds564) [reply siriushlafac/as_ap_trajfit/main.py:146]: right! Thanks!

Comment (fernandohds564) [reply siriushlafac/as_ap_trajfit/main.py]: done!

**Review by murilobalves — APPROVED** [review]

**Review by xresende — APPROVED** [review]

**Pull request merged**